tests/Media/ImageCollectionTest.php

```diff
@@ -7,9 +7,11 @@
 use CultuurNet\UDB3\Language;
 use CultuurNet\UDB3\Media\Properties\Description;
 use CultuurNet\UDB3\Media\Properties\MIMEType;
+use CultuurNet\UDB3\Model\ValueObject\Identity\UUID;
 use CultuurNet\UDB3\Model\ValueObject\MediaObject\CopyrightHolder;
 use PHPUnit\Framework\TestCase;
-use ValueObjects\Identity\UUID;
+use Ramsey\Uuid\Uuid as BaseUUID;
+use ValueObjects\Identity\UUID as LegacyUUID;
```

Comment:
I don't think this `LegacyUUID` is needed. You can just use a hardcoded value. Same applies for `BaseUUID`, you can just create a few uuids and use their string values.